### ISSUE TITLE
Refactor tidy patches into plugins

### ISSUE THREAD

**Issue by elieux** [Alexpux/MSYS2-pacman#6]
I see there are some big patches for the tidying phase of makepkg. Now that the [phase is extendable](https://lists.archlinux.org/pipermail/pacman-dev/2015-February/019884.html), maybe some of it should be put into separate files for better maintainability?

**Comment by Alexpux** [comment]
it already put. See develop branch. I'm build pacman with this changes when I test it.

**Comment by elieux** [comment]
Yeah, I know that you already merged the change into msys2-pacman. I meant that the strip plugin is now very big and does various stuff not related to stripping, for example handling executable files without .exe extension (https://github.com/Alexpux/MSYS2-pacman/blob/develop/scripts/libmakepkg/tidy/strip.sh.in#L86-101). Maybe this part of the strip plugin could now be a separate plugin.

**Comment by Alexpux** [comment]
i don't want to split this code because it hard to maintain it. this procedures a get from cygwin build system.

**Comment by Alexpux** [comment]
https://sourceforge.net/p/cygwin-ports/cygport/ci/master/tree/lib/src_postinst.cygpart

Search for "__prepstrip"
